core/src/test/kotlin/org/stellar/anchor/sep24/Sep24ServiceTest.kt

```diff
@@ -150,19 +148,17 @@ internal class Sep24ServiceTest {
     every { moreInfoUrlConstructor.construct(any(), any()) } returns
       "${TEST_SEP24_MORE_INFO_URL}?lang=en&token=$strToken"
     every { clientService.getClientConfigByDomain(any()) } returns
-      NonCustodialClientConfig(
-        "reference",
-        setOf("wallet-server:8092"),
-        "http://wallet-server:8092/callbacks"
-      )
+      NonCustodialClientConfig.builder()
+        .name("reference")
+        .domains(setOf("wallet-server:8092"))
+        .callbackUrl("http://wallet-server:8092/callbacks")
```

Comment:
How about migrate to use `callbackUrls` ince the `callbackUrl` is already marked as deprecated?

Reply:
I plan to migrate the usage in the merge PR